modules/keyvault/secret-autorotation/main.bicep

```diff
@@ -0,0 +1,348 @@
+
+@description('Optional. True if vnet integration should be enabled. If set to false, the vnet related parameters will be ignored. Default to false.')
+param isEnableVnet bool = false
+
+@description('Optional. Name of the subnet to be assigned to function app. Leave empty if private network is not used.')
+param functionAppSubnetId string = ''
+
+@description('Optional. True if the file share needs to be created in the storage account, this file share will host the function app files. The function app name will be used as the file share name. Default to true. NOTE: This seems required if using deployment script to provision the contents of the function app, if the fileshare does not exist at the time of creation, the deployment will fail.')
+param isCreateFileShare bool = true
+
+@description('Optional. Name of the file share to be created, default to functionAppName.')
+param fileShareName string = functionAppName
+
+@description('Optional. True if this module should assigned the role defined by param \'storageRoleId\' to the function app identity. Supports user assigned identity only. Default is false.')
+param isAssignStorageRole bool = false
+
+@description('Optional. Id of role that should be assigned to the function identity for the storage account. Default to storage account contributor role ID.')
+param storageRoleId string = resourceId('Microsoft.Authorization/roleDefinitions', '17d1049b-9a84-46fb-8f53-869881c3d3ab')
+
+@description('Optional. True if the storage on which the function will be created is accessible only from private network(vnet). Default to false.')
+param isStoragePrivate bool = false
+
+@description('Optional. Function app identity type. Default is SystemAssigned, which means the identity created with the function app will be used.')
+@allowed([
+  'UserAssigned'
+  'SystemAssigned'
+])
+param functionAppIdentityType string = 'SystemAssigned'
+
+@description('Optional. Name of the user assigned identity used to execute the deployment script for uploading function app source code. Must be provided if NOT using MSDeploy option. If user assigned identity is chosen for the function app, this will also be the identity used. Only effective when \'functionAppIdentityType\' is set to \'UserAssigned\'.')
+param userAssignedIdentityName string = ''
+
+@description('Optional. Resource group name of the user assigned identity, default to current resource group.')
+param userAssignedIdentityRg string = resourceGroup().name
+
+@description('Optional. Storage account name to be used by the deployment scripts. Deployment script by default create a temporary storage account during its execution but it is also possible to assign an existing storage instead.')
+param deploymentScriptStorage string = ''
+
+@description('Optional. Resource group of the deployment script storage, if \'deploymentScriptStorage\', this param will be ignored. Default to current resource group.')
```

Comment:
```suggestion
@description('Optional. Resource group of the deployment script storage, if `deploymentScriptStorage`, this param will be ignored. Default to current resource group.')
```

Reply:
Using backtick (`) instead of \' should provide a nicer appearance in the README.md